website/screens/components/select/specs/SelectSpecsPage.tsx

```diff
@@ -143,1056 +350,842 @@ const sections = [
+                  </td>
+                  <td>List dialog</td>
+                  <td>
+                    <Code>spacing-8</Code>
+                  </td>
+                  <td>0.5rem / 8px</td>
+                </tr>
+                <tr>
+                  <td>
+                    <Code>margin-top</Code>
+                  </td>
+                  <td>List dialog content</td>
+                  <td>
+                    <Code>spacing-4</Code>
+                  </td>
+                  <td>0.25rem / 4px</td>
+                </tr>
+                <tr>
+                  <td>
+                    <Code>margin-bottom</Code>
+                  </td>
+                  <td>List dialog content</td>
+                  <td>
+                    <Code>spacing-4</Code>
+                  </td>
+                  <td>0.25rem / 4px</td>
+                </tr>
+                <tr>
+                  <td>
+                    <Code>padding-top</Code>
+                  </td>
+                  <td>List option</td>
+                  <td>
+                    <Code>spacing-4</Code>
+                  </td>
+                  <td>0.25rem / 4px</td>
+                </tr>
+                <tr>
+                  <td>
+                    <Code>padding-bottom</Code>
+                  </td>
+                  <td>List option</td>
+                  <td>
+                    <Code>spacing-4</Code>
+                  </td>
+                  <td>0.25rem / 4px</td>
+                </tr>
+              </tbody>
+            </DxcTable>
+          </>
+        ),
+      },
+      {
+        title: "Size",
+        content: (
+          <>
+            <DxcParagraph>
+              The component <Code>width</Code> can adopt the following values:
+            </DxcParagraph>
+            <DxcTable>
+              <thead>
+                <tr>
+                  <th>Width</th>
+                  <th>Value</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td>
+                    <Code>small</Code>
+                  </td>
+                  <td>240px</td>
+                </tr>
+                <tr>
+                  <td>
+                    <Code>medium</Code> (<em>default</em>)
+                  </td>
+                  <td>360px</td>
+                </tr>
+                <tr>
+                  <td>
+                    <Code>large</Code>
+                  </td>
+                  <td>480px</td>
+                </tr>
+                <tr>
+                  <td>
+                    <Code>fillParent</Code>
+                  </td>
+                  <td>100%</td>
+                </tr>
+              </tbody>
+            </DxcTable>
+          </>
+        ),
+      },
+      {
+        title: "Margin",
+        content: (
+          <>
+            <DxcParagraph>
+              Select <Code>margin</Code> can use the values:
+            </DxcParagraph>
```

Comment:
Same here for the `margin` explanation.

Reply:
I'm not so sure here. I'm going to remove it but the Margin section seems to not be very consistent through the components...

Reply:
As you can see in the specs of all the components, in any of them there is this explanation, only the explanation related to how the margin is applied. I'm fine with its removal.

Box:
![image](https://user-images.githubusercontent.com/10975289/196119833-9cb95583-dac7-4319-bfc3-d8f83e3c1569.png)

Card:
![image](https://user-images.githubusercontent.com/10975289/196119937-e451e8d0-f2bc-41dc-b4fe-289c8d66d300.png)

Checkbox:
![image](https://user-images.githubusercontent.com/10975289/196120002-c5a4db5a-442f-48f9-be28-39f4125f80b8.png)

Dropdown:
![image](https://user-images.githubusercontent.com/10975289/196120122-fc227baa-1b6d-4181-9420-3b57c1a8039b.png)

Header:
![image](https://user-images.githubusercontent.com/10975289/196120234-c82542b1-86d8-483b-944b-5f01c89699c5.png)

Link:
![image](https://user-images.githubusercontent.com/10975289/196120342-515411bd-881a-43e0-8850-ee00221b2206.png)

Progress Bar:
![image](https://user-images.githubusercontent.com/10975289/196120554-4096c8cf-7070-4406-9d75-0bb7751a5dae.png)

Radio Froup:
![image](https://user-images.githubusercontent.com/10975289/196120638-a550ed1c-35ce-4f90-b347-ba89ceb5323e.png)

Switch:
![image](https://user-images.githubusercontent.com/10975289/196120765-ffadf89b-81ee-453b-9889-3145c22d4912.png)
Text Input:
![image](https://user-images.githubusercontent.com/10975289/196120881-3e44833d-30f2-47a5-9e57-d8013612885b.png)

Toggle Group:
![image](https://user-images.githubusercontent.com/10975289/196120956-3112b51e-2aa6-4761-8006-b44aed5c12c6.png)

Wizard:
![image](https://user-images.githubusercontent.com/10975289/196121026-cee0bb63-f6b6-445a-a7d6-f795182b3d57.png)